services/graphql/serverless.yml

```diff
@@ -0,0 +1,48 @@
+service: graphql
+
+plugins:
+  - serverless-webpack
```

Comment:
Please add `serverless-stack-output` :)

Reply:
Could be an item for QA for each service having an API :)